node-util/sbin/oo-init-quota

```diff
@@ -66,7 +66,7 @@ function get_fstab_mount_options() {
 # get the current mount options for a given mount point
 function get_current_mount_options() {
     # options are the comma delimited string between parens
-    mount | egrep " $1 " | sed -e 's/^.*(// ; s/).*$// ' | sort -u
+    mount | egrep "\s+$1\s+" | sed -e 's/^.*(// ; s/).*$// ' | sort -u
```

Comment:
`mount` shouldn't output tabs, so this change shouldn't be necessary, but then again, it shouldn't hurt anything.